experimental/handoff-library/csharp_dotnetcore/Handoff.sln

```diff
@@ -0,0 +1,37 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
```

Comment:
Please add the Dir.Props file from our codebase, and use the same StyleCop rules we are using. Have all the warnings set to generate errors.

Reply:
I don't see dir.props file anywhere in the repo. Can you point me to a solution/project file that use it, and I'll follow that.

Reply:
Found BotBuilder-DotNet.ruleset from the SDK repo, will use that here